addons/hr_expense/tests/test_expenses.py

```diff
@@ -1179,3 +1179,58 @@ def test_expense_from_attachments(self):
         self.env['hr.expense'].create_expense_from_attachments(attachment.id)
         expense = self.env['hr.expense'].search([], order='id desc', limit=1)
         self.assertEqual(expense.account_id, product.property_account_expense_id, "The expense account should be the default one of the product")
+
+    def test_expense_product_update(self):
+        #pylint: disable=bad-whitespace
```

Comment:
A small docstring explaining why this new test is needed / what it does would be nice